package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "StaticMaps-gl",
+  "version": "0.0.1",
+  "description": "Static map rendering with mapbox-gl",
+  "keywords": [],
+  "main": "./index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mapbox/StaticMaps-gl.git"
+  },
+  "license": "BSD-2-Clause",
+  "dependencies": {
+    "@mapbox/mapbox-gl-native": "^4.1.0",
+    "@mapbox/sphericalmercator": "^1.1.0",
+    "@turf/turf": "^5.1.6",
```

Comment:
Since `bbox` is the only `turf` function used we could install that package by itself `@turf/bbox`